content/en/docs/topics/version_skew.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Helm Version Support Policy"
+description: "Describes Helm's patch release policy as well as the maximum version skew supported between Helm and Kubernetes."
+---
+
+This document describes the maximum version skew supported between Helm and Kubernetes.
+
+## Supported Versions
+
+Helm versions are expressed as `x.y.z`, where `x` is the major version, `y` is the minor version, and z is the patch
+version, following [Semantic Versioning](https://semver.org/spec/v2.0.0.html) terminology.
+
+The Helm project maintains a release branch for the most recent minor release. Applicable fixes, including security
+fixes, are cherry-picked into the release branch, depending on severity and feasibility. Patch releases are cut from
+that branch as needed. This decision is owned by the release maintainer.
+
+## Supported Version Skew
+
+When a new version of Helm is released, it is compiled against a particular minor version of Kubernetes. For example,
+Helm 3.0.0 interacts with Kubernetes using the Kubernetes 1.16.2 client, so it is compatible with Kubernetes 1.16.
+
+As of Helm 3, Helm is assumed to be compatible with `n-3` versions of Kubernetes it was compiled against. Due to
+Kubernetes' volitility as a newer project, Helm 2's support policy is slightly stricter, assuming to be compatible
```

Comment:
I think this would read better as:

"Due to Kubernetes' volitility as a newer project" --> "Due to Kubernetes' changes between minor versions"